packages/generator/src/v1.1/generateMetadataFile.test.ts

```diff
@@ -38,8 +38,10 @@ describe(generateMetadataFile, () => {
       "import type { OntologyDefinition } from '@osdk/api';
       import type { Ontology as ClientOntology } from '@osdk/legacy-client';
       import type { Objects } from './ontologyObjects';
+      import type { Actions } from './ontologyActions';
       import { Todo } from './objects/Todo';
       import { Person } from './objects/Person';
+      import { markTodoCompleted } from './actions/markTodoCompleted';
```

Comment:
Something to watch out for and track is to make sure we don't conflict with the imports